test/native/__mocks__/@wordpress/block-library.js

```diff
@@ -0,0 +1,5 @@
+// Make sure the classic block is registered.
+window.wp = window.wp || {};
+window.wp.oldEditor = {};
```

Comment:
Could we move this to the jest setup file? I have some doubts about using a mock as a wrapper for the real thing and hardcoding the path like that.

Reply:
I removed this file since it was a copy from the gutenberg unit tests on `test/unit/__mocks__/@wordpress/block-library.js`